components/NewsAndUpdatesModal.js

```diff
@@ -127,7 +132,12 @@ const NewsAndUpdatesModal = ({ onClose, ...modalProps }) => {
       <ModalBody>
         <Query
```

Comment:
Nothing wrong here, but just so you know there's an RFC about using hooks instead HOC [here](
https://github.com/opencollective/opencollective/blob/main/rfcs/011-graphql.md#with-react-functional-components-react-hooks).
Next time you implement new components it would be nice to use hooks.

Reply:
@kewitz : Thanks for the heads up. Ben mentioned this to me as well; I'll keep this in mind. 😄